.nvmrc

```diff
@@ -0,0 +1 @@
+v15.3.0
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon: 
![image](https://user-images.githubusercontent.com/10209180/125988991-87ce523d-e23c-4b71-a0a2-b113d6975f52.png)

Reply:
in which case is this necessary? or always?

Reply:
all files should have blank line as the end of file